webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ArchiveApi.scala

```diff
@@ -1054,6 +1055,50 @@ trait SaveArchiveService {
   def save(archive: PolicyArchive, mergePolicy: MergePolicy, actor: EventActor): IOResult[Unit]
 }
 
+final case class TechniqueArchiveMod(added: Chunk[ResourceFile], updated: Chunk[ResourceFile])
```

Comment:
This case class does not seem to be used :thinking:

Reply:
you're right, forgot to delete it (it was before a simplification)